user-guide-chapters/introduction/building.adoc

```diff
@@ -15,76 +84,93 @@ https://go.dev/learn/[these instructions] to install the `golang` toolchain.  Af
 installation, make sure that the `path` environment variable or your IDE can find
 the toolchain.
 
```

Comment:
We should call out "Binary Image Flavors" - 1) "minimal" which builds only a minimal dist-spec conformant zot in order to reduce attack surface by the way of compiling only a minimal set of code and libraries, and 2) "extended" which builds the full zot will all extensions.

Reply:
Added image flavor descriptions.